### PR TITLE
add item presentation

### PR DESCRIPTION
**Pull request by ByJacob**

### CODE REVIEW

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -3,4 +3,10 @@
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
     <dimen name="lang_flag">40dp</dimen>
+    <dimen name="presentation_item_height">72dp</dimen>
+    <dimen name="presentation_item_padding">16dp</dimen>
+    <dimen name="presentation_item_text_padding">72dp</dimen>
+    <dimen name="presentation_item_text_large">16sp</dimen>
```
Comment (MBratkowski) [inline app/src/main/res/values/dimens.xml]: For text dimens you should use more generics name, for large text `text_large`. But now this is optional ;).

File: app/src/main/res/layout/item_presentation.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (MBratkowski) [inline app/src/main/res/layout/item_presentation.xml]: When You add new icon or different image resource you must start name with first char resource. Example:

- `example_icon_lang.png` - bad;
- `ic_presentation.png` - good;

File: app/src/main/res/values/dimens.xml (hunk 1 above)
Comment (ByJacob) [inline app/src/main/res/values/dimens.xml]: We don't need more item presentation, and don't need more textView in this item, don't change this name`

File: app/build.gradle [hunk 3]
```diff
@@ -60,6 +60,7 @@ dependencies {
     final JUNIT_VERSION = '4.12'
     final BUTTERKNIFE_VERSION = '8.4.0'
     final GSON_VERSION = '2.8.0'
+    final CONSTRAINT_LAYOUT_VERSION = '1.0.0-alpha9'
```
Comment (MBratkowski) [inline app/build.gradle]: Now actually Constraint layout library is available in version `1.0.0-beta3`. Check now version in your Android Studio. Open `SDK Manager`, click `SDK Tools`, expand the last position `Support Repository` and then check`Show pages details`, finally you have a list all version Constraint Layout in your manager.

File: app/src/main/res/layout/item_presentation.xml (hunk 2 above)
Comment (MBratkowski) [inline app/src/main/res/layout/item_presentation.xml]: Done ;).

**Pull request merged**